mllib/src/main/scala/org/apache/spark/ml/classification/Classifier.scala

```diff
@@ -74,7 +74,7 @@ abstract class Classifier[
    *                 and features (`Vector`).
    * @param numClasses  Number of classes label can take.  Labels must be integers in the range
    *                    [0, numClasses).
-   * @throws SparkException  if any label is not an integer >= 0
+   * @note Throws `SparkException` if any label is not an integer is greater than or equal to 0
```

Comment:
`is not a nonnegative integer`? http://mathworld.wolfram.com/NonnegativeInteger.html